Felienne/hedy
# Adds option to quiz question to show output container

**Description**
In this PR we make a differentiation between `code` and `output` snippets in the quiz question. Currently we only support either a textual question or a textual question in combination with a code snippet. We expand on this by also support a textual question in combination with an output snippet. The YAML is adjusted to support the `output` attribute as wel as the Typescript coding showing this. Currently this is only fixed for the English YAML, Weblate should take care of the rest.

**Fixes**
This PR fixes #2941.

**How to test**
Make sure your set language is English. Navigate to level 8 and start the quiz. Verify that question 6 contains an output example, without any syntax highlighting (as this is simply a plain `span` element). This approach should work identical to the code input/output approach of the main editor.
